ipa_room_exploration/package.xml

```diff
@@ -29,6 +29,7 @@
 	<depend>geometry_msgs</depend>
 	<depend>ipa_building_msgs</depend>
 	<depend>ipa_building_navigation</depend>
+        <depend>ipa_navigation_utils</depend>
```

Comment:
I think this package is public? We have the exploration software public and free for academic use. So we should either add a compile flag for using ipa_navigation or you need to make the ipa_navigation_utils public.